lib/backend/src/BackEnd.cpp

```diff
@@ -124,9 +124,48 @@ class AddWindowsDLLExportPass: public PassInfoMixin<AddWindowsDLLExportPass>
 	};
 };
 
+struct mlir::rlc::TargetInfoImpl
```

Comment:
i know that this is probably due to the rebase, but does this piece of code really need to be moved away from its original location?